### ISSUE TITLE
Dropdown and Autocomplete are unusable with screenreader after 13.2.0 accessibility improvements.

### ISSUE THREAD

**Issue by StefanCardnell** [primefaces/primeng#11213]
**I'm submitting a ...**  (check one with "x")
```
[X] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```
**Current behavior**

1) Download a free screenreader like NVDA (Windows - I am using this) or ORCA.
2) Go to the autocomplete or dropdown components on the showcase:

https://www.primefaces.org/primeng/showcase/#/autocomplete
https://www.primefaces.org/primeng/showcase/#/dropdown

3) Use keyboard tab to go over the components, and select options using the keyboard (space + up/down arrows).
4) Listen to the screen reader output.
5) The screenreader has the following problems:

- Screenreader doesn't mention the collapsible list exists when focusing on the component (autocomplete only).
- Screenreader doesn't mention the list opens (autocomplete + dropdown)
- When iterating over the list options, the screenreader doesn't announce the currently highlighted option (autocomplete + dropdown).

**Expected behavior**
 
Screenreader should do the things listed above.

**What is the motivation / use case for changing the behavior?**

I raised two issues illustrating this but they were closed with the note that they would be fixed with accessibility improvements:

https://github.com/primefaces/primeng/issues/11053
https://github.com/primefaces/primeng/issues/11054

However 13.2.0 released with accessibility improvements for autocomplete/dropdown but there are still issues with these components for users with screenreaders.

I can see in the latest commit to `p-autocomplete` many aria attributes [were simply just taken out](https://github.com/primefaces/primeng/commit/885de3a5df395a95a4a62eb1106454956e9fc92b). Taking out these attributes might help accessibility testing tools pass the autocomplete but the component is now unusable with a screen reader.

I kindly request that a screenreader tool like NVDA or Orca is downloaded and tested against these components - the accessibility issues become immediately clear. **The suggestions laid out in the two Github tickets above fix these problems and link the specifications which indicate how these issues should be solved with the correct aria attributes**.

(Also, I can see `p-dropdown` trigger was given a direct `aria-label="dropdown trigger"` - this will be an issue for non English users):

![image](https://user-images.githubusercontent.com/11324477/154355118-e64a8b6c-148e-42f5-a9fa-3a074abeb219.png)

**Please tell us about your environment:**

* **Angular version:** 13.2.2

* **PrimeNG version:** 13.2.0

* **Browser:** [Chrome  latest]
